data-gen/snoopAuditlog.py

```diff
@@ -157,12 +209,12 @@ def generate_coverage_report(openapi_spec, audit_log):
 
 def usage_and_exit():
   print("Usage:")
-  print("  snoopAuditlog.py <auditfile> <branch_or_tag> <outfile>")
+  print("  snoopAuditlog.py <auditfile> <outfolder>")
```

Comment:
snoopAuditlog.py now detects the branch / commit in metadata.json in order to pull the correct swagger.json, no need to provide it on the command line